api/desecapi/tests/test_stop_abuse.py

```diff
@@ -0,0 +1,54 @@
+from django.core import management
+
+from desecapi import models
+from desecapi.tests.base import DomainOwnerTestCase
+
+
+class StopAbuseCommandTest(DomainOwnerTestCase):
+
+    @classmethod
+    def setUpTestDataWithPdns(cls):
+        super().setUpTestDataWithPdns()
+        cls.create_rr_set(cls.my_domains[1], ['127.0.0.1', '127.0.1.1'], type='A', ttl=123)
+        cls.create_rr_set(cls.other_domains[1], ['40.1.1.1', '40.2.2.2'], type='A', ttl=456)
+        for d in cls.my_domains + cls.other_domains:
+            cls.create_rr_set(d, ['ns1.example.', 'ns2.example.'], type='NS', ttl=456)
+            cls.create_rr_set(d, ['ns1.example.', 'ns2.example.'], type='NS', ttl=456, subname='subname')
+            cls.create_rr_set(d, ['"foo"'], type='TXT', ttl=456)
+
+    def test_noop(self):
+        # test implicit by absence assertPdnsRequests
```

Comment:
Sure? (Otherwise, `assertPdnsNoRequestsBut` with empty argument.)

Reply:
Yes, sure, by virtue of having `DomainOwnerTestCase` as parent class